tests/features/rhdm/rhdm-kieserver.feature

```diff
@@ -31,3 +31,27 @@ Feature: RHDM KIE Server configuration tests
   Scenario: Check rhdm-kieserver extensions
     When container is ready
     Then container log should contain -Dorg.jbpm.server.ext.disabled=true -Dorg.jbpm.ui.server.ext.disabled=true -Dorg.jbpm.case.server.ext.disabled=true
+
+  Scenario: Check KIE_SERVER_JBPM_CLUSTER flag enabled
+    When container is started with env
+      | variable                        | value                |
+      | JGROUPS_PING_PROTOCOL           | kubernetes.KUBE_PING |
+      | KIE_SERVER_JBPM_CLUSTER         | true                 |
+    Then container log should contain KIE_SERVER_JBPM_CLUSTER enabled
+     And file /opt/eap/standalone/configuration/standalone-openshift.xml should contain <cache-container name="jbpm">
```

Comment:
what about the other items added on the xml?

Reply:
added